hierarchy-tool/tool/src/webapp/WEB-INF/velocity/show.vm

```diff
@@ -3,91 +3,217 @@
 
 #chef_portletContainer()
 #chef_portletContent()
+<style>
+    table, th, td {
+        border-collapse: collapse;
+    }
+    th, td {
+        padding: 5px;
+        text-align: left;
+        vertical-align:bottom;
+    }
+    .label_h3 {
+        color: #555;
+        padding: 0;
+        font-weight: bold;
+        font-family: 'PT Sans', 'Helvetica Neue', Arial, sans-serif;
```

Comment:
It's normally not a good idea to change the fonts inside Sakai, there should be "standard sakai fonts" that come from tool.css.

If each tool has it's own custom fonts then when someone changes them in the generic skin some parts of Sakai don't update.